src/opnsense/www/themes/opnsense/assets/stylesheets/main.scss

```diff
@@ -478,6 +478,7 @@ a.list-group-item:focus {
   text-decoration: none !important;
   color: #ec6d12 !important;
   background-color: lightergrey !important;
+  border: none! important; 
```

Comment:
@marjohn56 shouldn't this be `none !important` ? de space looks misplaced.

Reply:
My fault